CIP-0036/test-vector.md

```diff
@@ -1,110 +1,159 @@
-# Test vector for CIP15
+# Test Vector for CIP-0036
 
-### Inputs
+## Keys
 
-Payment **public** key hex
+**Note:** Not all keys are required for certificate recreation.
+
+Payment **private** signing key:
+```
+{
+    "type": "PaymentSigningKeyShelley_ed25519",
+    "description": "Payment Signing Key",
+    "cborHex": "5820614fdfe13d403bee2014570b190d81851f17d8daca0b6dd1ce33014403191003"
+}
```

Comment:
Can we go back to hex-only notation? Or, have a CIP eventually that standardize the cardano-cli text-envelope format. For now, this format has been breaking compatibility with itself every other release, so it's likely that these test vectors will be invalid in a month. 

In contrast, raw hex-encoded bytestring is forever-compatible and interoperable with anything 👍

Reply:
> Can we go back to hex-only notation?
Yes I agree - change made.